src/test/java/codeu/model/store/basic/UserStoreTest.java

```diff
@@ -141,6 +141,64 @@ public void testUpdateUserDescription_updateFailed() {
         .updateUserDescription(USER_ONE.getName(), newDescription);
   }
 
+  @Test
+  public void testUpdateUserPassword_UserNotRegistered() {
```

Comment:
For this file: use the 4 space indentation like other methods in this class

Reply:
You can use the google style tool for intellij that Kevin mentioned earlier then press cmd+option+l to format highlighted code.